Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -54,17 +54,9 @@ extension File {
 }
 
 public enum AccessControlLevel: String {
-    case Private = "private"
```

Comment:
It may be better to keep your changes but add explicit conformance to `CustomStringConvertible` and declare `var description: String` with the short/lowercase member names.